internal/mempool/mempool.go

```diff
@@ -218,10 +218,6 @@ type Policy struct {
 // additional metadata.
 type TxDesc struct {
```

Comment:
Since `TxDesc` is a wrapper of `mining.TxDesc`  now and not providing any additional information would it be worth returning the wrapped type in this PR? This should require a version bum as well since the modifications are happening in the internal package, thoughts?

Reply:
My goal is ultimately to reverse the dependency.  It should be that `mining` relies on `mempool`, not the other way around,

Since both of the packages are internal, they are no longer separate modules, so they don't have any module versions to bump.  That is one of the major benefits of internal packages.  Unless you're referring to something else?

Reply:
Oops, noticed a typo in my comment that I just fixed. Was referring to not needing to bump versions with the change I was talking about. We currently have the `TxDesc` as:
```go
type TxDesc struct {
	mining.TxDesc
}
```

As it is now we could remove this wrapper and use `mining.TxDesc` where `TxDesc` is.  This is a suggestion to clean things up some more, might not be worth it if dependency is going to change. Will let you to decide.

Reply:
Got it.  It's a good suggesting and ordinarily I'd agree, though I think I'd like to leave it in this case because the plan is to ultimately move what is defined in `mining.TxDesc` to be directly defined in `mempool.TxDesc` and then reverse the dependencies so `mining` relies on `mempool`, which more realistically models the actual relationship.  I say that because every node has a mempool and relays transactions regardless of whether or not it also has mining template generation enabled.

Reply:
Gotcha 👍.